x-pack/plugin/profiling/src/main/java/org/elasticsearch/xpack/profiling/CloudProviders.java

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.profiling;
+
+import java.util.Map;
+
+import static java.util.Map.entry;
+
+public class CloudProviders {
```

Comment:
In order to avoid that this class is unintentionally extended, can you please declare it as `final`? Also, to avoid that is unintentionally instantiated, can you please declare a private default constructor? (see class `IndexAllocation` for an example of both techniques)

Reply:
Thanks, done.